src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -184,26 +184,6 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
                                 session.getFeedbackSessionName());
         verifyEmail(email, instructor1.email, subject, "/sessionSubmissionConfirmationEmailZeroTimeZone.html");
 
-        ______TS("no email alerts sent for sessions not answerable/viewable for students");
```

Comment:
This test takes a private session to testing. But I am not sure if it is directly related to private sessions or not. (I assume it is). So, want you to confirm that this test needs deletion.

Reply:
From reading the `TS` description, I believe needs to be a test case for sessions with no questions that are answerable for students. Can only remove this if such a test case exists elsewhere.

Reply:
Will have to add another session in the json file, and also blocked because of #8761

Reply:
No longer blocked :)